apps/src/templates/manageStudents/ManageStudentsAgeCell.jsx

```diff
@@ -1,19 +1,25 @@
 import React, {Component, PropTypes} from 'react';
 import {ages} from '../AgeDropdown';
+import {connect} from 'react-redux';
+import {editAge} from './manageStudentsRedux';
 
 class ManageStudentAgeCell extends Component {
   static propTypes = {
     id: PropTypes.number.isRequired,
     age: PropTypes.number,
     isEditing: PropTypes.bool,
+    // Provided by redux
+    editAge: PropTypes.func.isRequired,
   };
 
   state = {
     ageValue: this.props.age
   };
 
   onChangeAge = (e) => {
-    this.setState({ageValue: e.target.value});
+    const newAge = e.target.value;
+    this.setState({ageValue: newAge});
+    this.props.editAge(this.props.id, newAge);
```

Comment:
line 21 is for internal state - used to display changes.
line 22 is saved to redux, so that when we hit "save" all the info to send to the server is up to date.

is it weird to have these separate? should they be combined? what's the performance implications of updating redux on every edit?

@islemaster before I write tests, do you mind giving me some feedback on the general strategy of this PR?

Reply:
It is a little weird to have these separate.  You've got two sources of truth on the student age, although I see in this case they're slightly different - `props.age` and redux always hold the saved age, while `state.ageValue` holds a potentially mutated, unsaved value.  I don't think it's necessary here, and you could just update redux and remove `state` from this component.  There shouldn't be a negative performance implication.